### ISSUE TITLE
Option not showing on iOS 12.4

### ISSUE THREAD

**Issue by tallyxx** [iAlex11/PasteAndGo2#1]
Tallyxx7 from Reddit here.
About the Past and Go 2 issue on ios 12

What info do u need?

**Comment by tallyxx** [comment]
Oh and about the browsers 

There was neither an option on safari or chrome

**Comment by alexmodrono** [comment]
I’m working on a fix for the iOS 12 and older version. Would you mind if I send you the .deb directly or prefer to install it from a repo through Cydia or Sileo?

**Comment by alexmodrono** [comment]
To avoid debugging using NSExceptions, I'll make a preference bundle that will allow you to debug the tweak easily.

I think I found the cause of the problem.

**Comment by tallyxx** [comment]
I wouldn’t mind either the deb file or it being on a Cydia repo

**Comment by tallyxx** [comment]
I’ll be around in the evening (in 7hours; at 6PM CET if you’d like to test it then.


Also looking forward, if you wanna test anything for 12.4, I’m always down as I don’t plan to update in the foreseeable future

**Comment by alexmodrono** [comment]
Cool! I'll try to have the fix ready by that hour.

**Comment by tallyxx** [comment]
Hoi, I’m around now, ready when u are

**Comment by alexmodrono** [comment]
Hey! I'm still working on the preference bundle, sorry about that. I'll tell you when the fix is ready.
It seems people are having problems with a tweak called **Sliepnizer** causing the option not to show.
Do you have it installed?

**Comment by tallyxx** [comment]
I can install n check it out

**Comment by tallyxx** [comment]
Oh if u meant that whether I have it installed and it conflicts with PasteAndGo2? Then no I don’t have it

**Comment by tallyxx** [comment]
Were you able to make any progress so far?

**Comment by alexmodrono** [comment]
Yeah, I'm working on the preference bundle and I think I fixed the issue. 
I'm working on the new features right now and will make an update once I finish them. 

Sorry for the inconvenience

**Comment by tallyxx** [comment]
Hey great to hear!

And there was no inconvenience at all; keep it up and looking forward to this and future releases

**Comment by alexmodrono** [comment]
Hey! Sorry @tallyxx for the wait, but please know the fix' release is gonna take more time since I'm moving PasteAndGo 2 from Twickd to Dynastic.
